src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -264,7 +278,9 @@ def __replace_restricted_symbols(input_string:str):
             input_string = input_string.replace(symbol, '*')
         return input_string
 
-    def __get_test_list_by_runtime(self):
+    def __get_test_list_by_runtime(self, test_unit = "test"):
+        self._total_test_cnt = 0
+        self._disabled_tests.clear()
```

Comment:
`self._disabled_tests` is empty here :)

Reply:
for the first time only, but we call this function 3 times